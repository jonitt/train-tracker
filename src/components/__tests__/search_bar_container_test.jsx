import { shallow } from "enzyme";
import React from "react";
import SearchBarContainer from "../search_bar/search_bar_container.jsx";

describe("SearchBarContainer", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<SearchBarContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
