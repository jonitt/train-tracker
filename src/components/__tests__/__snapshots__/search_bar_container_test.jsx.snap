// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBarContainer renders correctly 1`] = `ShallowWrapper {}`;
